comp/forwarder/defaultforwarder/forwarder.go

```diff
@@ -18,16 +19,33 @@ type dependencies struct {
 	Params Params
 }
 
-func newForwarder(dep dependencies) Component {
+type provides struct {
+	fx.Out
+
+	Comp           Component
+	StatusProvider status.InformationProvider
+}
+
+func newForwarder(dep dependencies) provides {
 	return NewForwarder(dep.Config, dep.Log, dep.Params)
 }
 
 //nolint:revive // TODO(ASC) Fix revive linter
-func NewForwarder(config config.Component, log log.Component, params Params) Component {
+func NewForwarder(config config.Component, log log.Component, params Params) provides {
 	if params.UseNoopForwarder {
-		return NoopForwarder{}
+		return provides{
+			Comp: NoopForwarder{},
+			StatusProvider: status.NewInformationProvider(statusProvider{
```

Comment:
We need to add a noop status provider 😄 but for fast prototype I used the same in both cases